static/usage/v7/datetime/styling/calendar-days/angular/example_component_html.md

```diff
@@ -0,0 +1,3 @@
+```html
+<ion-datetime presentation="date" [(ngModel)]="datetime"></ion-datetime>
```

Comment:
Let's add a comment as mentioned in Sean's [comment](https://github.com/ionic-team/ionic-docs/pull/3101#issuecomment-1696566830)